js/scripts.js

```diff
@@ -0,0 +1,46 @@
+// Jacob here, found something which might work.
```

Comment:
This, if left for another developer in the team might be better to leave as notes on the Trello board or in conversation versus in the code. Reduce excess comment clutter.